lighthouse-core/scripts/build-report-to-dist.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
```

Comment:
rename file? dist destination seems unimportant but building a *sample_v2* report (and maybe the fact that it's for `now`) does seem important